Sample/SampleForageSDK/Sections/CardNumber/CardNumberView.swift

```diff
@@ -319,6 +336,15 @@ class CardNumberView: UIView {
             padding: UIEdgeInsets(top: 24, left: 24, bottom: 0, right: 24)
         )
         
+        errorLabel.anchor(
+            top: reusableLabel.safeAreaLayoutGuide.bottomAnchor,
+            leading: contentView.safeAreaLayoutGuide.leadingAnchor,
+            bottom: nil,
+            trailing: contentView.safeAreaLayoutGuide.trailingAnchor,
+            centerXAnchor: contentView.centerXAnchor,
+            padding: UIEdgeInsets(top: 24, left: 24, bottom: 0, right: 24)
+        )
+        
```

Comment:
Were these updates to support QA tests and/or to help with local development? Good idea! I should take a hint from this in Android

Reply:
yep, both local dev and qa tests^